src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.DefaultItems.props

```diff
@@ -0,0 +1,44 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.Web.DefaultItems.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0">
+
+  <ItemGroup Condition="'$(EnableDefaultItems)' == 'true' 
+             And '$(EnableDefaultContentItems)' == 'true' 
+             And '$(EnableDefaultNoneItems)' == 'true'
+             And '$(AppDesignerFolder)' != ''">
```

Comment:
I wonder when AppDesignerFolder gets set. If a user sets it in the project file will this still work?

Reply:
It is defined in the property in NetSdk. So this would be always available and users can also override from csproj through a property is needed.

Reply:
I understand that but I wonder if this code still works if the user changes it in the csproj. It might be worth trying

Reply:
Discussed with Bill. This is because Razor still sets this initially in the Razor SDK - https://github.com/dotnet/aspnetcore/blob/master/src/Razor/Microsoft.NET.Sdk.Razor/src/Sdk/Sdk.Razor.StaticAssets.ProjectSystem.props#L49